cmd/firewall_remove.go

```diff
@@ -182,3 +173,11 @@ func prepareDefaultRemove(sg *egoscale.SecurityGroup, ruleName string, rule *ego
 	}
 	return nil, fmt.Errorf("missing rule %q", ruleName)
 }
+
+func init() {
+	firewallRemoveCmd.Flags().BoolP("force", "f", false, cmdFlagForceHelp)
+	firewallRemoveCmd.Flags().BoolP("ipv6", "6", false, "Remove rule with any IPv6 source")
+	firewallRemoveCmd.Flags().BoolP("my-ip", "m", false, "Remove rule with my IP as a source")
+	firewallRemoveCmd.Flags().BoolP("all", "", false, "Remove all rules")
```

Comment:
I don't know if it was on purpose, but in any case the use of `BoolP()` without passing a short flag instead of `Bool()` was wrong ¯\\\_(ツ)_/¯ Thanks for spotting this!